extensions/markdown/README.md

```diff
@@ -0,0 +1,30 @@
+## Using Markdown-it Plugins
+
+To use a `markdown-it` plugin, goto the directory containing the `markdown-it` extension.  This will
+be found in the `extensions` folder in your vscode installation.  Install the `markdown-it` plugin
+as a node module.  For example, to install the `KaTeX` markdown-it extension you would simply type:
+
+``` sh
+npm install markdown-it-katex
+```
+
+Next, add the plugin into your [settings configuration](https://code.visualstudio.com/docs/customization/userandworkspace)
+under the `"markdown.plugs"` namespace.  This setting should contain a list for each `markdown-it` extension you wish
+to install.  The list should contain an object describing the name of the module, a list of styles that your extentsion
+may require, and options that you wish to pass to your extension.  For example, while installing the `markdown-it-katex`
+extension we would include this in your settings configuration:
+
+``` json
+    "markdown.plugins": [
+        {
+            "name": "markdown-it-katex",
+            "styles": [
```

Comment:
Sorry for the delayed response.  Do you mean to ask if plugins need stylesheets or if it's necessary to keep the stylesheet references with the plugin declarations in `markdown.plugins`? KaTeX for instance does need stylesheets, whether or not these should be declared here of in `markdown.styles` may be up to debate.  I personally like having styles that are associate with plugins with their plugin declarations themselves, but I guess I don't mind either way.

I liked the idea of having a default folder for where we should lookup styles, which I opted for the 'media' folder in the Markdown extension.  Using `markdown.styles` when providing a single filename like "styles.css", it appears that VSCode will only look in the root directory of the current project.  This certainly isn't ideal for plugins, so the alternative would be to either provide a URL or a full filepath.  As a user installing a markdown plugin, where do you think is a good place to put stylesheets for markdown plugins?  I guess a more general question would be: where is a good place to configuration files for vscode extensions?  Has this been done?  Maybe we could do whatever they do for consistency?